source/Ship.cpp

```diff
@@ -2710,6 +2710,22 @@ int Ship::Scan(const PlayerInfo &player)
 	if(isYours || (target->isYours && activeScanning))
 		Audio::Play(Audio::Get("scan"), Position());
 
+	bool isImportant;
```

Comment:
Always initialise primitives.
```suggestion
	bool isImportant = false;
```